libs/components/text-editor/src/lib/modules/text-editor/url-modal/text-editor-url-modal.component.ts

```diff
@@ -89,8 +89,8 @@ export class SkyTextEditorUrlModalComponent {
     }
   }
 
-  public activeTabChanged(value: number): void {
-    this.activeTab = value;
+  public activeTabChanged(value: string | number): void {
+    this.activeTab = Number(value);
```

Comment:
Was this to fix a bug?

Reply:
Yes, I was seeing an error locally. I've updated to use `SkyTabIndex` directly (which is an alias for `string | number`).